### PR TITLE
Add Elasticsearch alerts remedy

### PR DESCRIPTION
**Pull request by raihankhan**
Signed-off-by: raihankhan <raihan@appscode.com>

### CODE REVIEW

File: charts/elasticsearch/remedy.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  - Contact AppsCode team
+- #### ElasticsearchHeapUsageWarning
+  - Get the current `heap.percent` for each node using the [_cat/nodes](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-nodes.html) API
+  - Delete indices with old or unnecessary data. Eliminate empty indices. Keep the number of aggregation buckets (size) in your queries to a minimum. Try reducing the size of the large bulk index requests.
+  - [Reconfigure](https://kubedb.com/docs/v2022.08.08/guides/elasticsearch/configuration/jvm-options/) the default [JVM options](https://www.elastic.co/guide/en/elasticsearch/reference/current/advanced-configuration.html#set-jvm-heap-size) as per your requirement
+  - Contact AppsCode team
+- #### ElasticsearchDiskOutOfSpace
+  - Reconfigure storage capacity to your cluster -
+    - [Horizontally Scale](https://kubedb.com/docs/v2022.08.08/guides/elasticsearch/concepts/elasticsearch-ops-request/#spechorizontalscaling) up your data nodes with `ElasticsearchOpsRequest`
```
Comment (spectro30) [inline charts/elasticsearch/remedy.md:16]: Shouldn't it be volume expansion?

File: charts/elasticsearch/remedy.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    - [Vertically Scale](https://kubedb.com/docs/v2022.08.08/guides/elasticsearch/concepts/elasticsearch-ops-request/#specverticalscaling) up your data nodes with `ElasticsearchOpsRequest`
+  - [Reduce replicas per shard](https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html)
+  - Contact AppsCode team
+- #### ElasticsearchDiskSpaceLow
+  - Reconfigure storage capacity to your cluster -
```
Comment (spectro30) [inline charts/elasticsearch/remedy.md:21]: Shouldn't it be volume expansion?

File: charts/elasticsearch/remedy.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  - Cluster pending tasks are updates to the cluster state which may have been initiated directly by a user or by the cluster itself. View the pending cluster tasks using [_cluster/pending_tasks](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-pending.html#cluster-pending) API.
+- #### ElasticsearchNoNewDocuments10M
+  - Indicates the cluster haven't received any new documents for the past 10 minutes!
+
+### KubeDB Provisioner
+
+- #### AppPhaseNotReady
+  - Contact AppsCode team
+- #### AppPhaseCritical
+  - If any `MariaDBOpsRequest` is ongoing on same database, Wait until it completes.
```
Comment (spectro30) [inline charts/elasticsearch/remedy.md]: `ElasticsearchOpsRequest`

File: charts/elasticsearch/remedy.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  - If some nodes of the MariaDB Cluster are not `Up`, Try restarting those nodes one at a time.
```
Comment (spectro30) [inline charts/elasticsearch/remedy.md]: `Elasticsearch` Cluster or similar

File: charts/elasticsearch/remedy.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  - Contact AppsCode team if this persists for more than 30 minutes.
+
+### KubeDB OpsManager
+
+- #### OpsRequestOnProgress
+  - Just a reminder, nothing to worry about.
+- #### OpsRequestStatusProgressingToLong
+  - If any `MariaDBSQLOpsRequest` is ongoing on same database, Wait until it completes.
```
Comment (spectro30) [inline charts/elasticsearch/remedy.md]: `ElasticsearchOpsRequest`

File: charts/elasticsearch/remedy.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+  - Contact AppsCode team
+- #### RepositoryCorrupted
+  - Check if the `INTEGRITY` of `repository` is true
+  - Contact AppsCode team
+- #### RepositoryStorageRunningLow
+  - Increase the volume size of `repository` backend
+  - Update RetentionPolicy of the BackupConfiguration to free up storage
+- #### BackupSessionPeriodTooLong | RestoreSessionPeriodTooLong
+  - Check if the `INTEGRITY` of `repository` is true
+  - Check the `MariaDB` CRs status
```
Comment (spectro30) [inline charts/elasticsearch/remedy.md]: `Elasticsearch`

**Review by spectro30 — CHANGES_REQUESTED** [review]

File: charts/elasticsearch/remedy.md (hunk 2 above)
Comment (raihankhan) [reply charts/elasticsearch/remedy.md:19]: yes

File: charts/elasticsearch/remedy.md (hunk 1 above)
Comment (raihankhan) [reply charts/elasticsearch/remedy.md]: yes

**Review by spectro30 — APPROVED** [review]
LGTM

**Pull request merged**